docs/design/rules-api.md

```diff
@@ -0,0 +1,111 @@
+# Rules API
+
+Observatorium is featured with a multi-tenant API that enables tenants to write and read Prometheus recording and alerting rules.
+
+The API is [defined](https://github.com/observatorium/api/blob/main/rules/spec.yaml) using the [OpenAPI](https://swagger.io/specification/) specification format.
+
+The goal is to enable tenants to create, modify and access their own rules.
+
+## Usage
+
+### Create rules
+
+```bash
```

Comment:
The code block below is indeed she'll script, however this one is not. Maybe we should remove this language annotation